chain/beacon/beacon.go

```diff
@@ -25,7 +25,9 @@ type Schedule []BeaconPoint
 func (bs Schedule) BeaconForEpoch(e abi.ChainEpoch) RandomBeacon {
 	for i := len(bs) - 1; i >= 0; i-- {
 		bp := bs[i]
-		if e >= bp.Start {
+		// The new network version kicks in at one _after_ the upgrade height itself,
+		// so we switch to the new beacon when _strictly_ greater than bp.Start
+		if e > bp.Start {
```

Comment:
Won't this break existing migration point?

Reply:
Oh, you're right, I forgot that we upgraded from incentinet to mainnet. I'll see how best to fix that up.

Reply:
As long as we switch the same in miner, no issue. It's mostly just code/system smell.

Reply:
I like that, I'm actually inclined to switch _later_ than the upgrade height (get your migration and other nonsense done first, then we switch beacons).

Reply:
I'm just thinking we may want to ship two upgrades at the same time instead of trying to bundle them into one.